src/screens/BookDetail/styles.module.scss

```
@@ -0,0 +1,47 @@
+@import 'variables/colors';
+
+.book {
+  border: 1px solid $gray-main;
+  box-shadow: 0 0 13px 0 rgba(50, 50, 50, 0.91);
```

use scss variables insisde rgba functio 
```suggestion
  box-shadow: 0 0 13px 0 rgba($main-shadow, 0.91);
```